components/scream/src/share/tests/common_physics_functions_tests.cpp

```diff
@@ -326,6 +347,7 @@ void run(std::mt19937_64& engine)
   for (int k=0; k<num_mid_packs; ++k) {
 
     // Make sure all results don't contain invalid numbers
+    REQUIRE( Check::is_non_negative(density_host(k),k) );
```

Comment:
Anything else you'd like to test here? Is it worth checking anything based on the pseudo-density?

Reply:
Nothing comes to mind, the density function is based on pseudo_density and I put in a comparison to the ideal-gas-law formulation up above to check consistency that way.